install/helm/space-agon/values.yaml

```diff
@@ -21,18 +21,20 @@ frontend:
   image: 
     repository: local/space-agon-frontend
     tag: TAG
-    imagePullPolicy: IfNotPresent
+    imagePullPolicy: Always
+  omCoreAddress: "https://om-core-976869741551.us-central1.run.app"
 
 mmf:
   name: mmf
   replicas: 1
   containerPort: 50502
-  serviceType: ClusterIP
+  serviceType: LoadBalancer
   servicePort: 50502
   image: 
     repository: local/space-agon-mmf
     tag: TAG
-  imagePullPolicy: IfNotPresent
+  imagePullPolicy: Always
+  ipAddress: 10.128.0.30
```

Comment:
When we deploy the space-agon on public cloud such as GKE, it is difficult to specify the ip address in advance.  

Is this IP address used in the Open Match 2 ?

Reply:
Indeed on GKE, by default the IP address is random. However, you can reserve an IP address for your GKE cluster and use that reserved IP address for the service. By doing so, the service will always be turned up with that specified IP address. I outlined the reasons and steps in the PR description as well.

Reply:
Correction: it's not "reserve an IP address for your GKE cluster", it's "reserve an IP address in your vpc network" by going into "VPC Network" page, then "IP addresses" tab and then "RESERVE INTERNAL STATIC IP ADDRESS"